recipes/bbmap/meta.yaml

```diff
@@ -1,6 +1,7 @@
 {% set name = "BBMap" %}
-{% set version = "38.22" %}
-{% set sha256 = "7ee6165440efc5fcfd108416d041e00c7d89002bc1fabfeb74a9c175528c9dea" %}
+{% set version = "38.44" %}
+{% set sha1 = "d66ba0802a191f5f63ef05148bb5bff6e09fcdd9" %}
+{% set md5 = "723c3d6a0d0a81aa4901c1d1e5e4d883" %}
 
```

Comment:
Just use one of the hashes both here and in the `source:` section.

Reply:
Okay! I wasn't sure which was preferred as SourceForge only had the sha1 and md5 checksums. Changed in e1c8fd00101d6150cffa605c16f87b98b504a3b0